peer_test.go

```diff
@@ -22,17 +22,26 @@ package tchannel_test
 
 import (
 	"fmt"
+
 	"runtime"
+
```

Comment:
There's blank lines between standard library imports which is not right. goimports might be misconfigured or something, not sure but you should check and make sure that the import groups are correct.

Make this import look like:
```go
import (
  "fmt"
  "runtime"
  "sort"
  "sync"
  [... and other standard library imports]

  . "github.com/uber/tchannel-go"

  [all other imports without any blank lines]
)
```